authors/apps/authentication/urls.py

```diff
@@ -10,4 +10,6 @@
     path('user/<int:user_id>', UserRetrieveUpdateAPIView.as_view(), name='current_user'),
     path('users/signup/', RegistrationAPIView.as_view(), name='register'),
     path('users/login/', LoginAPIView.as_view(), name='login'),
+    path('accounts/forget_password/', ForgetPasswordAPIView.as_view(), name='forgot'),
```

Comment:
@mikenthiwa in the spirit of uniformity please use `/forgot_password` in the URL.

Reply:
@mikenthiwa the change of the word forget to forgot is still pending.